tests/image.js

```diff
@@ -66,8 +66,9 @@ describe( 'Image', () => {
 			setModelData( model, '[<image alt="alt text" src="/assets/sample.png"></image>]' );
 
 			expect( getViewData( view ) ).to.equal(
-				'[<figure class="ck-widget ck-widget_selected image" contenteditable="false">' +
+				'[<figure class="ck-widget ck-widget_selected ck-widget_with-resizer image" contenteditable="false">' +
```

Comment:
All those changes in this file can be reverted after you change https://github.com/ckeditor/ckeditor5-image/pull/298/files#r312947942.